Prox/Prox/PlaceDetails/PlaceDetailViewController.swift

```diff
@@ -66,10 +66,10 @@ class PlaceDetailViewController: UIViewController {
         return panGesture
     }()
     
-    lazy var mapButton: MapButton = {
-        let button = MapButton()
+    lazy var mapButton: UIButton = {
+        let button = UIButton()
+        button.setImage(#imageLiteral(resourceName: "button_map"), for: .normal)
```

Comment:
`#imageLiteral` shows up automagically in Xcode, with the image inlined directly in the source (you don't actually see this `#imageLiteral` if you look at the source in Xcode).